Sources/Sentry/Public/SentryOptions.h

```diff
@@ -8,6 +8,8 @@ NS_ASSUME_NONNULL_BEGIN
 NS_SWIFT_NAME(Options)
 @interface SentryOptions : NSObject
 
+
```

Comment:
h: We need description of this new property like the others.

Reply:
Updated description.